proto/internal/uber/cadence/admin/v1/service.proto

```diff
@@ -112,6 +112,12 @@ service AdminAPI {
 
   // ListDynamicConfig returns all dynamic config parameters and associated values.
   rpc ListDynamicConfig(ListDynamicConfigRequest) returns (ListDynamicConfigResponse);
+
+  // DeleteWorkflow is used to delete a workflow with given domainName, workflowID and runID
+  rpc DeleteWorkflow(AdminDeleteWorkflowRequest) returns (DeleteWorkflowResponse);
+
+  // MaintainCorruptWorkflow is used to delete a workflow with given domainName, workflowID and runID
```

Comment:
Looking at the comment alone this seems identical to `DeleteWorkflow`. If it is different maybe mention if the comment how it it is different?